Gemfile

```diff
@@ -40,7 +40,7 @@ gem "haml"
 gem "haml-rails"
 
 gem 'devise', '~> 4.6.0'
-gem 'omniauth-google-oauth2'
+gem 'omniauth-google-oauth2', '<= 1.9.0'
```

Comment:
The security alert is telling us that versions older than or equal to `1.9.0` are vulnerable. Therefore, we should be using a version of `omniauth` that is greater than `1.9.0`.

Reply:
oh

Reply:
There isn't any version of omniauth-google-oauth2 that passes 1.9.0. Actually, its previous version was 0.6.0, with its current version being 0.7.0.

Reply:
sad